### PR TITLE
Making getIrodsResponseStatusCode() a utility file

### PR DESCRIPTION
**Pull request by sam-i-am012**
Addresses #54

### CODE REVIEW

File: src/test/java/org/example/IrodsResponseUtils.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package org.example;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+
+
+
```
Comment (korydraughn) [inline src/test/java/org/example/IrodsResponseUtils.java:9]: Collapse to one empty line.

**Review by korydraughn — COMMENTED** [review]
Just one comment to resolve.

File: src/test/java/org/example/IrodsResponseUtils.java (hunk 1 above)
Comment (sam-i-am012) [reply src/test/java/org/example/IrodsResponseUtils.java]: Done.

**Comment by sam-i-am012** [comment]
Done.

**Comment by korydraughn** [comment]
You can squash the code review commit into the original commits.

**Review by korydraughn — APPROVED** [review]
Looks good. Pound it.

**Comment by sam-i-am012** [comment]
Done.

**Comment by korydraughn** [comment]
Merging.

**Pull request merged**